src/app/CommandPathParams.h

```diff
@@ -30,6 +30,11 @@ enum class CommandPathFlags : uint8_t
     kGroupIdValid    = 0x02,
 };
 
+/**
+ *
+ * @brief This is used to record command path, and can pass command path info between IM and ember library.
```

Comment:
```suggestion
 * @brief This is used to record a command path.
```
Why would one "record a command path"?